script/push-l10n.py

```diff
@@ -26,7 +31,15 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description='Push strings to Transifex')
```

Comment:
Need to update description.

Reply:
Fixed in [13140e2](https://github.com/brave/brave-core/pull/27005/commits/13140e28ea7cd53e4056067f06938b892d409c16)